nuget/AppCenterPush.nuspec

```diff
@@ -20,7 +20,7 @@
       </group>
       <group targetFramework="MonoAndroid403">
```

Comment:
Should we update `targetFramework` here? I remember we already discussed it in the past and the answer was "no".
@guperrot Do you remember details?

Reply:
This is the minimum target or something like that to select the right dlls instead of .net standard.
It's unrelated to this PR anyway but we might need to use the correct string for API level 16.